packages/modular-scripts/src/build/index.ts

```diff
@@ -129,6 +131,20 @@ async function buildApp(target: string) {
     }
   }
 
+  // Add dependencies from source and bundled dependencies to target package.json
+  const packageDependencies = await getPackageDependencies(target);
+  const targetPackageJson = (await fs.readJSON(
```

Comment:
I would suggest for an app that we generate a "minimal" `package.json` content instead of pulling everything directly across. Similar to how we do for `package` types.

Reply:
Done, limiting the fields in the public package.json